Scripts/utils.py

```diff
@@ -213,3 +213,15 @@ def print_error_title_fetch():
   print(f"  > You won't be able to delete/hide any comments like usual, but you can {F.LIGHTMAGENTA_EX}exclude users before saving the log file{S.R}")
   print(f"  > Then, you can {F.LIGHTGREEN_EX}delete the comments later{S.R} using the {F.YELLOW}mode that removes comments using a pre-existing log file{S.R}")
   input("\n Press Enter to continue...")
+
+def clear_terminal() -> None:
```

Comment:
Is there a reason you have made this function way more complicated than it needs to be? For example why use `sys.stdout.write("\033[2J\033[1;1H")` when you could just do `os.system("clear")`.

Reply:
I'd think sys.stdout.write() (practically just print) would be at least a tiny bit faster than os.system()

Reply:
But no significant difference other than os.system() runs a command and sys.stdout.write() does not

Reply:
It's faster according to here https://www.geeksforgeeks.org/clear-console-c-language/

Reply:
As for "more complicated", there are only two `if`s in clear_terminal():

if it is not a tty (ex. the script is running as a GUI application or is being piped into another program or file), do not clear anything and return

if windows: os.system(cls)
else: write "\033[2J\033[1;1H"

As a bonus, `clear_command` is no longer a variable, and therefore no arbitrary os.system() call could be made by changing `clear_command`; clear_terminal() is easier to read 😁

Reply:
Oh also, in the future, a config option could be added to disable clearing the terminal (probably dev_mode or something) by either redefining clear_terminal() or adding some check to clear_terminal()